frontend/src/components/TaskBoard.css

```
@@ -0,0 +1,215 @@
+    padding: 10px;
+  }
+}
+
+/* Task view controls styling */
+.task-view-controls {
+  margin-bottom: 16px;
+  padding: 0 16px;
+  background-color: #f3f4f6;
+  border: 1px solid #d1d5db;
+  border-radius: 8px;
+  padding: 12px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)